tools/packaging/static-build/cache_components.sh

```diff
@@ -39,14 +40,22 @@ cache_kernel_artifacts() {
 	local kernel_tarball_name="kata-static-cc-kernel.tar.xz"
 	local current_kernel_image="$(get_kernel_image_name)"
 	local current_kernel_version="$(get_from_kata_deps "assets.kernel.version")"
-	if [ -n "${TEE}" ]; then 
+	if [ -n "${TEE}" ]; then
 		kernel_tarball_name="kata-static-cc-${TEE}-kernel.tar.xz"
 		[ "${TEE}" == "tdx" ] && current_kernel_version="$(get_from_kata_deps "assets.kernel.${TEE}.tag")"
 		[ "${TEE}" == "sev" ] && current_kernel_version="$(get_from_kata_deps "assets.kernel.${TEE}.version")"
 	fi
 	create_cache_asset "${kernel_tarball_name}" "${current_kernel_version}" "${current_kernel_image}"
 }
 
+cache_virtiofsd_artifacts() {
+	local virtiofsd_tarball_name="kata-static-cc-virtiofsd.tar.xz"
+	local current_virtiofsd_version="$(get_from_kata_deps "assets.externals.virtiofsd.version")-$(get_from_kata_deps "assets.externals.virtiofsd.toolchain")"
```

Comment:
Nice!

Can you update https://github.com/kata-containers/kata-containers/blob/CCv0/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh#L243 to match this one?